controllers/imagecollector/imagecollector_controller.go

```diff
@@ -389,6 +394,19 @@ func (r *Reconciler) createScanJob(ctx context.Context, collector *eraserv1alpha
 		},
 	}
 
+	resources := &scanJob.Spec.Template.Spec.Containers[0].Resources
```

Comment:
why are we handling these differently?

Reply:
@sozercan because they have no default values associated (related to the above)